ansible/roles/datadog/templates/datadog.conf.j2

```diff
@@ -16,7 +16,7 @@ gce_updated_hostname: yes
 use_mount: no
 
 {% if inventory_hostname in groups['proxy'] %}
-dogstreams: {{ log_home }}/{{ deploy_env }}-timing.log:{{ datadog_parsers_dest }}/nginx/timings.py:parse_nginx_timings
+dogstreams: {{ log_home }}/{{ deploy_env }}-timing.log:{{ datadog_parsers_dest }}/nginx/timings.py:parse_nginx_timings, {{ log_home }}/{{ deploy_env }}-timing.log:{{ datadog_parsers_dest }}/nginx/timings.py:parse_nginx_apdex
```

Comment:
This is gonna get unmanageable real fast! Might be worth having a datadog/vars.yml that has these listed like

```yaml
dogstreams:
  - input: {{ log_home }}/{{ deploy_env }}-timing.log
    python_file: {{ datadog_parsers_dest }}/nginx/timings.py
    python_function: parse_nginx_timings
  - input: {{ log_home }}/{{ deploy_env }}-timing.log
    python_file: {{ datadog_parsers_dest }}/nginx/timings.py
    python_function: parse_nginx_apdex
```